.travis.yml

```diff
@@ -23,6 +23,20 @@ script:
 
 jobs:
   include:
+    # Python 3.7 in Travis: https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
```

Comment:
Can you add PIP=8.1.1, 9.0.1 and 9.0.3 to the py3.7 matrix please.
Let's keep appveyor/travis matrix as close as possible.